src/components/Table/TableFormatters/DetailFormatter.vue

```diff
@@ -110,14 +110,15 @@ export default {
       return resource.replace(' details', '').replace('详情', '')
     },
     getDrawerTitle() {
-      if (this.formatterArgs?.getDrawerTitle && typeof this.formatterArgs.getDrawerTitle === 'function') {
-        return this.formatterArgs.getDrawerTitle({
+      let drawerTitle = ''
+      if (this.formatterArgs?.getTitle && typeof this.formatterArgs.getTitle === 'function') {
+        drawerTitle = this.formatterArgs.getTitle({
           col: this.col,
           row: this.row,
           cellValue: this.cellValue
         })
       }
-      let title = this.cellValue || this.row.name
+      let title = this.cellValue?.name || drawerTitle
       const resource = this.getResource()
       if (resource) {
         title = `${resource}: ${title}`
```

Comment:
The code contains one typo which is `details` instead of `'details'`. The function to set the title should be defined inside the method that handles getting the title based on formatter arguments, not as a separate function:

```
if (this.formatterArgs?.getTitle && typeof this.formatterArgs.getTitle === 'function') {
    this.setDrawerTitle();
}

// SetDrawerTitle logic here

return { title };
```

Please review these updates before proceeding with any changes.